fladle-plugin/src/main/java/com/osacky/flank/gradle/FulladlePlugin.kt

```diff
@@ -138,3 +45,127 @@ class FulladlePlugin : Plugin<Project> {
     }
   }
 }
+
+fun configureLibraryModule(project: Project, flankGradleExtension: FlankGradleExtension) = project.run {
+  pluginManager.withPlugin("com.android.library") {
```

Comment:
is it possible to rely on this logic only and remove the `plugins.findPlugin("com.android.application")` line?

Reply:
It is possible, `configureLibraryModule` won't do anything when an app module is passed. But I think adding those `findPlugin` checks makes line 28-31 more readable, It makes it obvious that library modules are passed to the `configureLibraryModule` and doesn't assume the reader knows that it doesn't matter if the check doesn't happen.